include/libtorrent/alert_types.hpp

```diff
@@ -1843,7 +1843,7 @@ namespace libtorrent
 		// interpret these values throughout the process' runtime.
 		//
 		// For more information, see the session-statistics_ section.
-		std::uint64_t values[counters::num_counters];
+		std::int64_t values[counters::num_counters];
```

Comment:
ok, I think it may have relied on this at some point in the past, perhaps by swapping a std::vector<> out of the alert rather than copying.